src/preferences/dialog/dlgprefdeckdlg.ui

```diff
@@ -104,41 +115,34 @@
         </item>
        </layout>
       </item>
-      <item row="0" column="0">
-       <widget class="QLabel" name="labelCueMode">
+      <item row="2" column="0">
+       <widget class="QLabel" name="labelTimeDisplay">
         <property name="text">
-         <string>Cue mode</string>
+         <string>Time Format</string>
         </property>
-        <property name="openExternalLinks">
-         <bool>true</bool>
+       </widget>
+      </item>
+      <item row="2" column="1">
+       <widget class="QComboBox" name="comboBoxTimeFormat"/>
+      </item>
+      <item row="3" column="0">
+       <widget class="QLabel" name="labelAutoCue">
+        <property name="text">
+         <string>Auto cue</string>
         </property>
         <property name="buddy">
-         <cstring>ComboBoxCueMode</cstring>
+         <cstring>checkBoxSeekToCue</cstring>
         </property>
        </widget>
       </item>
-      <item row="0" column="1" colspan="2">
-       <widget class="QComboBox" name="ComboBoxCueMode">
+      <item row="3" column="1" colspan="2">
+       <widget class="QCheckBox" name="checkBoxSeekToCue">
         <property name="toolTip">
-         <string>Mixxx mode:
-- Cue button while pause at cue point = preview
-- Cue button while pause not at cue point = set cue point
-- Cue button while playing = pause at cue point
-Mixxx mode (no blinking):
-- Same as Mixxx mode but with no blinking indicators
-Pioneer mode:
-- Same as Mixxx mode with a flashing play button
-Denon mode:
-- Cue button at cue point = preview
-- Cue button not at cue point = pause at cue point
-- Play = set cue point
-Numark mode:
-- Same as Denon mode, but without a flashing play button
-CUP mode:
-- Cue button while pause at cue point = play after release
-- Cue button while pause not at cue point = set cue point and play after release
-- Cue button while playing = go to cue point and play after release
-</string>
+         <string>Automatically seeks to the first saved cue point on track load.
+            If none exists, seeks to the beginning of the track.</string>
```

Comment:
The leading space are included in the tooltip.

Reply:
done.